src/Microsoft.ML.DataView/VBuffer.cs

```diff
@@ -77,16 +117,27 @@ public VBuffer(int length, T[] values, int[] indices = null)
         }
 
         /// <summary>
-        /// Construct a possibly sparse representation.
+        /// Construct a possibly sparse vector representation.
         /// </summary>
+        /// <param name="length">The length of the constructed buffer.</param>
+        /// <param name="count">The count of explicit entries. This must be between 0 and <paramref name="count"/>, both
+        /// inclusive. If it equals <paramref name="length"/> the result is a dense vector, and if less this will be a
+        /// sparse vector.</param>
+        /// <param name="values">The values to be used. This must be at least as long as <paramref name="count"/>. If
+        /// <paramref name="count"/> is 0, it is legal for this to be <see langword="null"/>.</param>
+        /// <param name="indices">The values to be used. If we are constructing a dense representation, or
```

Comment:
`The values to be used.`  This must be a copy-paste error.  This is the `indices`.